Phase03/FullTextSearch/Controllers/Logic/DocumentsLoader/WordsListFormatFixer.cs

```diff
@@ -0,0 +1,17 @@
+using FullTextSearch.Controllers.Logic.Abstraction;
+
+namespace FullTextSearch.Controllers.Logic.DocumentsLoader;
+
+public static class WordsListFormatFixer
+{
+    public static IEnumerable<string> FixWordsList(this IEnumerable<string> documentWords,
+        List<IStringReformater> reformaters)
+    {
+        return documentWords.Select(w =>
+        {
+            foreach (var format in reformaters) w = format.FixWordFormat(w);
+
+            return w;
+        });
```

Comment:
do not modify input arguments (why?)
create a new list of fiexed words and return